src/pages/chainlink-automation/tutorials/nft-quickstart.mdx

```diff
@@ -0,0 +1,122 @@
+---
+layout: ../../../layouts/MainLayout.astro
+section: automation
+date: Last Modified
+title: "Batch NFT Reveal"
+whatsnext: { "FAQs": "/chainlink-automation/faqs" }
+---
+
+import { Aside, CodeSample, ClickToZoom } from "@components"
+import { Tabs } from "@components/Tabs"
+
+Using [Chainlink VRF](https://docs.chain.link/vrf/v2/introduction/) in generative art NFT collections is de-facto the standard approach for getting provably random source in smart contracts. By batching the reveal process, instead of making VRF calls for each NFT we can save cost up to 100x (in a collection of 10,000 with batch size of 100).
```

Comment:
```suggestion
Using [Chainlink VRF](https://docs.chain.link/vrf/v2/introduction/) in generative art NFT collections is the standard approach for getting provably random source in smart contracts. By batching the reveal process, instead of making VRF calls for each NFT we can save cost up to 100x (in a collection of 10,000 with batch size of 100).
```